GET_STARTED.md

```diff
@@ -0,0 +1,390 @@
+...
+2022-01-01T12:00:00.000000Z -0.000 0   [info ] ../src/node/node_state.h:1790        | Network TLS connections now accepted
+# It is then possible to interact with the service
+```
+
+Or, for the non-SGX (a.k.a. virtual) variant:
+
+```bash
+docker build -t ccf-app-template:cpp-virtual -f docker/ccf_app_cpp.virtual .
+docker run ccf-app-template:virtual
+```
+---
+
+# Network Governance
+a Consortium of trusted Members [governs the CCF network](https://microsoft.github.io/CCF/main/governance/index.html). members can submit proposals to CCF and these proposals are accepted based on the rules defined in the [Constitution](https://microsoft.github.io/CCF/main/governance/constitution.html).
+Governance changes are submitted to a [network as Proposals](https://microsoft.github.io/CCF/main/governance/proposals.html), and put to a vote from members.
+
+Submit a proposal 
+```bash
+proposal0_out=$(/opt/ccf/bin/scurl.sh "https://ccf_service_url/gov/proposals" --cacert service_cert.pem --signing-key member0_privk.pem --signing-cert member0_cert.pem --data-binary @proposal.json -H "content-type: application/json")
+proposal0_id=$( jq -r  '.proposal_id' <<< "${proposal0_out}" )
+```
+
+Members vote to accept or reject the proposal
+```bash
+/opt/ccf/bin/scurl.sh "https://ccf_service_url/gov/proposals/$proposal0_id/ballots" --cacert service_cert.pem --signing-key member0_privk.pem --signing-cert member0_cert.pem --data-binary @vote_accept.json -H "content-type: application/json" | jq
+/opt/ccf/bin/scurl.sh "https://ccf_service_url/gov/proposals/$proposal0_id/ballots" --cacert service_cert.pem --signing-key member1_privk.pem --signing-cert member1_cert.pem --data-binary @vote_accept.json -H "content-type: application/json" | jq
+```
+ 
+## Network Governance: Activating network members 
+
+By default the CCF network needs at least one member to be started, after the network is started this member must be activated.
+[Adding or activating members](https://microsoft.github.io/CCF/main/governance/adding_member.html)
+
+### Activate member
+```bash
+curl "https://ccf_service_url/gov/ack/update_state_digest" -X POST --cacert service_cert.pem --key member0_privk.pem --cert member0_cert.pem --silent | jq > request.json
+cat request.json
+/opt/ccf/bin/scurl.sh "https://ccf_service_url/gov/ack"  --cacert service_cert.pem --signing-key member0_privk.pem --signing-cert member0_cert.pem --header "Content-Type: application/json" --data-binary @request.json
+```
+
+### New member proposal
+```json
+{
+  "actions": [
+    {
+        "name": "set_member",
+        "args": {
+            "cert": "member_cert",
+            "encryption_pub_key": "member_encryption_pub_key"
+        }
+    }
+  ]
+}
+```
+
+## Network Governance: Adding users 
+Users are directly interact with the application running in CCF. Their public identity should be voted in by members before they are allowed to issue requests.
```

Comment:
Light edit: 

Users directly interact with the application running in CCF. Their public identities should be voted in by members before they are allowed to issue requests.

Reply:
Done